src/modules/components/proposal/proposalActions/proposalActionsAction/proposalActionsAction.tsx

```diff
@@ -0,0 +1,54 @@
+import { Accordion, Heading } from '../../../../../core';
+import { useOdsModulesContext } from '../../../odsModulesProvider';
+import { ProposalActionsActionVerification } from '../proposalActionsActionVerfication/proposalActionsActionVerfication';
+import type { IProposalAction } from '../proposalActionsTypes';
+import { proposalActionsUtils } from '../utils';
+
+export interface IProposalActionsActionProps {
+    /**
+     * Proposal action
+     */
+    action: IProposalAction;
+    /**
+     * Index of the action being mapped by its parent ProposalActions.Container
+     */
+    index: number;
+    /**
+     * Toggle accordion callback for notifying the parent container to update after click
+     */
+    onToggle?: () => void;
+}
+
+export const ProposalActionsAction: React.FC<IProposalActionsActionProps> = (props) => {
+    const { action, index, onToggle } = props;
+    const ActionComponent = proposalActionsUtils.getActionComponent(action);
+    const { copy } = useOdsModulesContext();
+
+    const actionTypeToStringMapping: Record<string, string> = {
+        withdrawToken: copy.proposalActionsAction.actionTypeWithdrawToken,
+    };
```

Comment:
Given that we also want to make the custom-actions customisable, it won't be possible to customise the action names in that way. What if we instead add an `actionNames` optional property (`Record<string, string>`)  to the container component and fallback to the function name when available?
It can be something like:
```
const actionNames = Object.keys(ActionTypes).reduce((accumulator, current) => ({ ...accumulator, [current]: t(`app.governance.action.${current}`), {});

<Action.Container actions={actions} actionNames={actionNames} />
```

The property can be then passed to the action component (if having the actions property set on the container) as:
<Action.Action action={action} name={actionNames?.[action.type]} />